src/Sulu/Component/Util/SuluNodeHelper.php

```diff
@@ -70,7 +70,7 @@ public function getLanguagesForNode(NodeInterface $node)
         $languages = array();
         foreach ($node->getProperties() as $property) {
             /** @var PropertyInterface $property */
-            preg_match('/^' . $this->languageNamespace . ':([a-zA-Z_]*?)-title/', $property->getName(), $matches);
+            preg_match('/^' . $this->languageNamespace . ':([a-zA-Z_]*?)-changer/', $property->getName(), $matches);
```

Comment:
well, it raises a question .. if the title is mandatorily translated, then it is no problem, otherwise we need another method to track the locales